dashboard/src/modules/containers/MainLayout/index.jsx

```diff
@@ -1,20 +1,48 @@
 import React from "react";
+import { Outlet, useLocation } from "react-router-dom";
 import ToastComponent from "modules/components/ToastNotificationComponent";
 import { useSelector } from "react-redux";
-import { Page, Spinner } from "@patternfly/react-core";
-import NavbarDrawer from "modules/components/NavbarDrawerComponent";
+import HeaderComponent from "modules/components/HeaderComponent";
+import {
+  Spinner,
+  Page,
+  PageSection,
+  PageSectionVariants,
```

Comment:
If we are going to alphabetize in other modules, we should alphabetize here.  I'd suggest making such an alphabetization change in a separate PR.

Reply:
Let's take it separately